packages/clerk-js/src/core/clerk.test.ts

```diff
@@ -152,6 +153,35 @@ describe('Clerk singleton', () => {
         expect(mockSession.touch).not.toHaveBeenCalled();
       });
     });
+
+    it('calls __unstable__onBeforeSetActive before session.touch', async () => {
+      mockSession.touch.mockReturnValueOnce(Promise.resolve());
```

Comment:
🙃 i think we should add a `expect.assertions(2)` to ensure that expectation in `__unstable__onBeforeSetActive` is called.